ads/teads.md

```diff
@@ -19,20 +19,17 @@ limitations under the License.
 ## Example
 
 ```html
-<amp-ad width=300 height=220
+<amp-ad width=300 height=1
```

Comment:
Is this working well for you?

Is `layout=responsive` working OK with such an extreme aspect ratio?